pennylane/__init__.py

```diff
@@ -23,6 +23,8 @@
 
 from semantic_version import Version, Spec
 
+# QueuingContext needs to be imported before all other pennylane imports
+from ._queuing_context import QueuingContext  # pylint: disable=wrong-import-order
```

Comment:
:+1:

Reply:
Out of curiosity: how come the order is set here like this?

Reply:
`QueuingContext` now acts as the baseclass for `BaseQNode` and `OperationRecorder`, therefore I have to import it before those classes are created, i.e. in the subsequent imports.